whynotearth/BrowTricks
# 301 add pmu form screen after signup

# Issue Being Addressed

#301 

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ ] Refactor
[x] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description

**For New Features:** What feature are we adding? Explain your technical approach to doing this.

Show a screen to force creating a form template.

# How to Test/Reproduce

Go to `/sign-up/business-info`  
Create a new tenant and see the new screen after signup and check the bottom navigation.

# Screenshots/casts

<img src="https://user-images.githubusercontent.com/510242/93610823-cc13db80-f9e2-11ea-83e3-071666a5ee3c.png" width="400" />
